### PR TITLE
Move DS's Kerberos env vars to unit file

### PR DESCRIPTION
**Pull request by tiran**
The IPA specific env vars KRB5_KTNAME and KRB5CCNAME are now defined in a instance specific ipa-env.conf unit file.
    
Fixes: https://pagure.io/freeipa/issue/7860

### CODE REVIEW

File: ipapython/ipautil.py [hunk 1]
```diff
@@ -1104,14 +1104,17 @@ def reverse_record_exists(ip_address):
     return True
 
 
-def config_replace_variables(filepath, replacevars=dict(), appendvars=dict()):
+def config_replace_variables(filepath, replacevars=dict(), appendvars=dict(),
+                             removevars=None):
     """
     Take a key=value based configuration file, and write new version
-    with certain values replaced or appended
+    with certain values replaced, appended, ir removed
```
Comment (abbra) [inline ipapython/ipautil.py]: Please fix the typo `ir removed`

Comment (tiran) [reply ipapython/ipautil.py]: - Restore from a different version is not supported. ipa-server-upgrade should fix the cause, though.
- I have added ``systemd_daemon_reload``.

**Comment by tiran** [comment]
- Restore from a different version is not supported. ipa-server-upgrade should fix the cause, though.
- I have added ``systemd_daemon_reload``.

**Comment by abbra** [comment]
LGTM. I'll wait for the remaining CI test before adding labels.

**Comment by tiran** [comment]
master:

* 3cb1ccb3b0a4f361d4d3ef667c60d5b6f4a23333 Add option to remove lines from a file
* e4621f1204c8ffc3e1279cea2a897292d560a48c Add tasks.systemd_daemon_reload()
* 1dfac4f5b7e9986e1217d6979b854b94de094aae Move DS's Kerberos env vars to unit file

**Pull request closed without merging**